### PR TITLE
Remove old b3dm/i3dm/pnts content classes

### PR DESCRIPTION
**Pull request by ptrgags**
As part of #10346, this PR removes `Batched3DModel3DTileContent`, `Instanced3DModel3DTileContent` and `PointCloud3DTileContent`. `Model3DTileContent` will be used instead.

A couple of notes:

* I also removed specs for b3dm classification, as #10623 will handle this
* I disabled the tests for `Cesium3DTileBatchTable` until we can consolidate that with `ModelFeatureTable`, since the tests used `b3dm` files which no longer use the old style of batch table (only `.vctr` and `.geom` do)

@j9liu could you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by ptrgags** [comment]
@j9liu I checked the Sandcastles and they're all good except for the ones with the BIM model (since that change is only on `main`). I'm going to merge `main -> replace-model` then `replace-model -> remove-1.0-contents` so we can better test those Sandcastles.

File: Specs/Cesium3DTilesTester.js [hunk 1]
```diff
@@ -193,6 +197,7 @@ Cesium3DTilesTester.rejectsReadyPromiseOnError = function (
       fail("should not resolve");
     })
     .catch(function (error) {
+      console.log(error);
```
Comment (j9liu) [inline Specs/Cesium3DTilesTester.js:200]: Was this added for debugging purposes, or is this an intentional change?

**Comment by ptrgags** [comment]
@j9liu okay, the Sandcastles with the BIM tileset should work correctly now!

File: Source/Scene/ModelExperimental/Model3DTileContent.js [hunk 2]
```diff
@@ -384,7 +393,9 @@ Model3DTileContent.fromGeoJson = function (tileset, tile, resource, geoJson) {
     content,
     additionalOptions
   );
-  content._model = ModelExperimental.fromGeoJson(modelOptions);
+  const model = ModelExperimental.fromGeoJson(modelOptions);
+  content._model = model;
+  content._readyPromise = model.readyPromise;
   return content;
```
Comment (j9liu) [inline Source/Scene/ModelExperimental/Model3DTileContent.js:399]:
```suggestion
  content._readyPromise = model.readyPromise;
  
  return content;
```
one more nitpick for consistency with the other `fromXXX` functions

**Review by j9liu — CHANGES_REQUESTED** [review]
@ptrgags looks good for the most part, just a few comments!

I'll run through the Sandcastles locally as well to make sure no new bugs came up

File: Specs/Cesium3DTilesTester.js (hunk 1 above)
Comment (ptrgags) [reply Specs/Cesium3DTilesTester.js:200]: oops forgot to remove it :sweat_smile:

**Comment by ptrgags** [comment]
@j9liu oops forgot to mention the PR is ready for review again!

**Comment by j9liu** [comment]
I found a regression but I don't think it has to hold up this PR. In the 3D Tiles Formats sandcastle, `PointCloudDraco` isn't properly responding to the "Debug Colorize" option in the inspector:
draco point

For reference, this is what it does on [sandcastle.com](https://sandcastle.cesium.com/?src=3D%20Tiles%20Formats.html&label=3D%20Tiles) right now:

draco point main

Everything else looks good. Thanks @ptrgags !

**Pull request merged**